_posts/0000-01-02-Genere-Pridi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)
![4q74zl](https://user-images.githubusercontent.com/24882488/102485783-fa479c80-405f-11eb-8391-ad749676b680.jpg)